GeeksCoreLibrary/Modules/GclReplacements/Services/StringReplacementsService.cs

```diff
@@ -100,6 +100,17 @@ public async Task<string> DoAllReplacementsAsync(string input, DataRow dataRow =
                 foreach (Match m in regex.Matches(input))
                 {
                     var value = m.Groups[1].Value;
+                    
+                    // Check if there are any valid formatter functions used in the variable and if so, use the variable name without the formatter as translation.
```

Comment:
One small note here: It's not a translation, so the comment here needs updating.